Bigtable/src/Connection/Grpc.php

```diff
@@ -371,47 +378,270 @@ public function readRow(array $args)
      */
     public function readRows(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $tableName = $this->pluck('tableName', $args);
+        $optionalArgs = [];
+        if (isset($args['options'])) {
```

Comment:
I don't see any need to nest these arguments inside `$args.options`. `ReadRowsRequest` doesn't have a concept of `options`. Why not just pass them at the top level of `$args`?

Instead of plucking out of `$args.options`, manipulating and pushing into `$optionalArgs`, just pluck out of `$args`, manipulate and put back into `$args`. Pass `$args` into the request instead of `$optionalArgs`:

```php
if (isset($args['rows'])) {
    $args['rows'] = $this->rowSetObject($args['rows']);
}

return $this->send([$this->bigtableClient, 'readRows'], [
    $tableName,
    $args,
    $this->addResourcePrefixHeader($args, $tableName)
]);
```